README.md

```diff
@@ -29,214 +40,54 @@ This Connector implements the [Data Connector Spec](https://github.com/hasura/nd
 
-Clone the repository and run the following commands to build the Docker image that can then be used for code generation.
-
-```
-cd ndc-open-api-lambda
+#### Step 2.2: Choose a port for the connector
 
-# build the docker image with the name `ndc-oas-lambda` and tag `latest`
-docker build -t ndc-oas-lambda:latest .
+The CLI will ask for a specific port to run the connector on. Choose a port that is not already in use or use the
+default suggested port.
 
-# get command documentation/help
-docker run --rm ndc-oas-lambda:latest update -h
+#### Step 2.3: Provide the env vars required for the connector
 
-# when running the code generation, the container will expect either of the following two things
-# 1. The OpenAPI document is mounted at `/etc/connector/` as `swagger.json`. OR
-# 2. The link to the OpenAPI document is provided via the env var `NDC_OAS_DOCUMENT_URI`
-# Please note that mounting a directory at /etc/connector/ is required in both cases, since that is where the code is generated
+Environment variables that can be used to configure the connector are listed under [Supported Environment Variables](./docs/documentation.md#supported-environment-variables)
 
-# **** START: Examples ****
-# run the code generation
-docker run --rm -v ./:/etc/connector/ -e NDC_OAS_DOCUMENT_URI=${url to open API document} ndc-oas-lambda:latest update
+## Step 3: Introspect the connector
 
-# with baseUrl
-docker run --rm -v ./:/etc/connector/ -e NDC_OAS_DOCUMENT_URI=${url to open API document} -e NDC_OAS_BASE_URL=http://demoapi.com/ ndc-oas-lambda:latest update
-
-# **** END: Examples ****
-
-# start the NodeJS Lambda Connector
-docker run --rm -p 8080:8080 -v ./:/etc/connector ndc-oas-lambda:latest
+```bash
+ddn connector introspect <connector-name>
 ```
 
-NOTE: You can also pass CLI flags with values to the Docker Container instead of environment variables
-
-### Using the CLI
+This will generate the required HML and TS files
 
-You can install the OpenAPI Connector as a CLI on your system. Please ensure you have NPM and Node 20+ installed. You can install and run the CLI using the following commands
+## Step 4: Add your resources
 
+```bash
+ddn connector-link add-resources <connector-name>
 ```
-cd ndc-open-api-lambda
-
-# install dependencies and then install the CLI
-npm i
-npm run install-bin
-
-# print help for update command
-ndc-oas-lambda update -h
 
-# **** Examples ****
-# run the update command to generate code
-ndc-oas-lambda update --open-api ${link/path to open api swagger document}
-# with headers
-ndc-oas-lambda update --open-api ${link/path to open api swagger document} -H key1=value -H key2=value2 -H keyN=valueN
-# with baseurl and headers
-ndc-oas-lambda update --open-api ${link/path to open api swagger document} -H key1=value -H key2=value2 -H keyN=valueN -b http://localhost:8081
-
-# start the NodeJS Lambda Connector
-npm run watch
-```
+This command will track all the indices in your Elasticsearch DB as Models
```

Comment:
Copy-🍝 ? 🙃

Reply:
yeah 😆 

Resovled in [ccf47a2](https://github.com/hasura/ndc-open-api-lambda/pull/48/commits/ccf47a2cc402379395570986f81fddc615c84f60)